pretend_file1.java

```diff
@@ -3,3 +3,11 @@
 java code original line 3 Edited DWS on desktop GBPC2185
 jave code original line 4 Edited MF on desktop GBPC2141
 jave code original line 5
+
+/**
+ * 
+ **/
+public interface pretend_fileq1 {
+  
+  double factorial(int n); 
```

Comment:
@bennevis  'n' is not very intuitive name, could you propose something more readable?